repository/repository.go

```diff
@@ -62,6 +62,7 @@ type GatewayAAT struct {
 	Version              string `json:"version"`
 }
 
+type GatewaySettingsInput = GatewaySettings
```

Comment:
Why do you need these Input structs?

Reply:
It's for the GQLgen in PUB so it will use these structs instead of the ones from the GraphQL schema. For some reason, gqlgen marks optional fields as pointers and it requires so much extra code to convert them so I'd rather just have them be regular structs.